frontend/src/screens/study-landing.tsx

```diff
@@ -18,20 +18,44 @@ import {
     Text,
     Title,
 } from '@mantine/core';
+import { IconCheck } from '@tabler/icons-react'
 import Markdown from 'react-markdown'
 import { useLearningPathStudies } from './learner/studies';
 import { CompactStudyCard } from '../components/study/compact-study-card';
+import { notifications } from '@mantine/notifications';
 
 export default function StudyLanding() {
     const env = useEnvironment()
-
     const study = useLoaderData() as ParticipantStudy
     const learningPathStudies = useLearningPathStudies(study?.learningPath)
+    const notificationShown = useRef(false)
```

Comment:
Is there a simpler way to do this without using `useRef`? I think the user will just land on this page once, you can probably change the useEffect below to just have an empty dependency array so that it only shows once? Let me know if that works, should simplify this a bit :) @divitcr7